ledger/participant-integration-api/src/main/scala/platform/store/backend/StorageBackend.scala

```diff
@@ -431,15 +432,12 @@ object UserManagementStorageBackend {
   case class DbUser(internalId: Int, domainUser: User)
 }
 
-object MeteringStorageBackend {
-  case class TransactionMetering(
-      applicationId: Ref.ApplicationId,
-      actionCount: Int,
-      meteringTimestamp: Timestamp,
-      ledgerOffset: Offset,
-  )
-}
-
 trait MeteringStorageBackend {
-  def entries(connection: Connection): Vector[MeteringStorageBackend.TransactionMetering]
+  def entries(connection: Connection): Vector[TransactionMetering]
```

Comment:
How about following rename:
entries -> allEntries
transactionMetering -> applicationEntries

Reply:
The `entries` method was only there to support testing.  Now we have `transactionMetering` this can be used in place so:

1. I have removed `entries`
2. I have not renamed `transactionMetering` as that selects from the `transaction_metering` table and returns `TransactionMetering` records so seems well named

Reply:
:+1: